lib/callbacks.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const fs = require('fs');
+const util = require('util');
+const faker = require('faker');
+let path = process.argv.slice(2)[0];
+let file = process.cwd()+`/data/${path}`;
```

Comment:
Because you've not put anything in your `index.js`, you're duplicating a lot of code here.